build/wlanstart.sh

```diff
@@ -23,6 +23,10 @@ CONTAINER_PID=$(docker inspect -f '{{.State.Pid}}' ${HOSTNAME})
 CONTAINER_IMAGE=$(docker inspect -f '{{.Config.Image}}' ${HOSTNAME})
 if [ -z ${INTERFACE} ]; then
   INTERFACE=$(docker run -t --privileged --net=host --pid=host --rm --entrypoint /bin/sh ${CONTAINER_IMAGE} -c "iw dev" | grep 'Interface' | awk 'NR==1{print $2}')
+  # Additional method to get network interface
+  if [ -z ${INTERFACE} ]; then
+    INTERFACE=$(docker run -t --privileged --net=host --pid=host --rm --entrypoint /bin/sh ${CONTAINER_IMAGE} -c "ls /sys/class/ieee80211/*/device/net")
```

Comment:
What's the meaning of `ieee80211` in `/sys/class/ieee80211/*/device/net`?

Reply:
it is the wifi standard